package.json

```
@@ -11,6 +11,7 @@
     "preupload": "npm run test",
     "upload": "node script/upload.js",
     "update-data-sources": "script/update-data-sources.sh",
+    "update-electron": "node script/dl-api.js",
```

Sorry to be such a pain about naming things, but do you think we could make these match? maybe something like

```suggestion
    "update-electron-apis": "node script/update-electron-apis.js",
```

---

Alternatively, you could just add `node script/update-electron-apis.js` to the `update-data-sources.sh` script instead of it being its own npm script.

---

ba155d205eaca02309a27ef3a533d21cbf593a83 🚀